drivers/serial/uart_nrfx_uart.c

```diff
@@ -395,6 +410,15 @@ static int uart_nrfx_init(struct device *dev)
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
 
+	/* Global variable: m_init_uart_event_txrdy is necessary to make
```

Comment:
This comment looks really confusing, IMHO. I don't get what the "correct value" is or shall be.